core-bundle/src/Resources/config/voters.yml

```diff
@@ -0,0 +1,10 @@
+services:
+    _defaults:
+        autoconfigure: true
+        autowire: true
```

Comment:
Since when do we use autowiring?

Reply:
![](https://user-images.githubusercontent.com/1192057/85722777-ed6e6f80-b6f2-11ea-9c07-4c5d528d8981.jpg)

Reply:
:popcorn:

Reply:
I have removed `autowiring: true` from the config files in bdaba16c253e3a415eee245ff1efbbce5d36ce1e. However, as I said before, I think this is _stupid_, because we still use autowiring by definition of using `ServiceSubscriberInterface` to lazy-load the services… 🤷

Reply:
Why did you add `autowire: false`? Just drop the setting entirely.